sdk/samples/sender_app.c

```diff
@@ -168,10 +81,18 @@ int main(int argc, char** argv)
     video_pixel_format pix_fmt = PIX_FMT_YUV422P_10BIT_LE;
     uint32_t frame_size = 0;
 
+    char audio_type[5] = DEFAULT_AUDIO_TYPE;
+    char audio_format[5] = DEFAULT_AUDIO_FORMAT;
+    char audio_sampling[3] = DEFAULT_AUDIO_SAMPLING;
+    char audio_ptime[6] = DEFAULT_AUDIO_PTIME;
+    char anc_type[5] = DEFAULT_ANC_TYPE;
+    char payload_codec[6] = DEFAULT_PAYLOAD_CODEC;
+    uint32_t audio_channels = DEFAULT_AUDIO_CHANNELS;
+
     mcm_conn_context* dp_ctx = NULL;
     mcm_conn_param param = { 0 };
     mcm_buffer* buf = NULL;
-    uint32_t total_num = 300;
+    uint32_t total_num = DEFAULT_TOTAL_NUM;
```

Comment:
There's no default defined in any readme. This was changed on purpose. 0 = do not limit, send until the end of file / stopped by the user. It has to be 0 for audio, otherwise the output is mumbled. We may change the code to mitigate that later.

From usage/README:
```text
-n, --number=<number_of_frames>         Total frame number to send (default = inf: 0)
```